eks-worker-al2.json

```diff
@@ -15,7 +15,7 @@
     "encrypted": "false",
     "kms_key_id": "",
     "cni_version": "v0.6.0",
```

Comment:
What kind of verifications do we need to do to bump `cni_version` as well? The latest release is `v0.7.4`.

Reply:
The latest released version from upstream is still [v0.6.0](https://github.com/containernetworking/cni/releases)

Reply:
There's a [v0.7.2-rc2](https://github.com/containernetworking/cni/releases/tag/v0.7.0-rc2) but it's not released yet.

Reply:
Ah, yikes. 

>v0.6.0 squeed released this on Aug 17, 2017 · 193 commits to master since this release

I got confused by the release called [CNI v0.7.4](https://github.com/containernetworking/plugins/releases/tag/v0.7.4) since they also have [v0.6.0](https://github.com/containernetworking/plugins/releases/tag/v0.6.0), but that was of course just the plugins repo.